frontend/src/pages/profile/components/user-achievements/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+.achievementBlock {
+  display: flex;
+  grid-area: achievements;
+  flex-direction: column;
+  align-items: center;
+  max-width: 270px;
+  padding: 20px;
+  text-align: center;
+}
+
+.achievementList {
+  grid-template-columns: repeat(var(--achievement-columns), 1fr);
+  margin-bottom: 30px;
+  font-size: 12px;
+  column-gap: 5px;
+}
+
+.achievementItem {
+  cursor: pointer;
+}
+
+.title {
+  margin-bottom: 30px;
+}
+
+.buttonShow {
+  margin-top: auto;
+  padding: 10px 20px;
+  color: var(--white);
+  background-color: var(--dark-green);
+}
+
+.achievementListModal {
+  --achievement-columns: 4;
+
+  gap: 40px;
+  max-height: 410px;
+  padding: 20px 50px;
+  overflow: auto;
+}
+
+.achievementListModal::-webkit-scrollbar {
```

Comment:
Maybe it would be better to create a scss mixins for styling a scrollbar (text editor has the same styles `frontend\src\libs\components\text-editor\styles.module.scss`) or place these styles globall

Reply:
[0674fdf](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/156/commits/0674fdf588fdfb71c149e203ac514b4f92691278)